content/applications/productivity/documents.rst

```diff
@@ -229,19 +229,14 @@ Share a workspace
 You can share a link to a workspace and allow users to :guilabel:`Download` its content or
 :guilabel:`Download and Upload` files to it.
 
-To do so, go to the left column of your dashboard. In the :guilabel:`Workspace` section, select the
-workspace to share, and possibly one or several tags that will be automatically added to the
-uploaded documents. Then, click the arrow next to the :guilabel:`Upload` button and
-:guilabel:`Share`.
+To do so, select the workspace to share in the left column of your dashboard and possibly one or
+several tags that will be automatically added to the uploaded documents. Then, click the arrow next
+to the :guilabel:`Upload` button and :guilabel:`Share`.
```

Comment:
```suggestion
To do so, go to the left column of your dashboard and select the workspace you want to share. You can also choose one or more tags that will be automatically added to the uploaded documents.. Then, click the arrow next
to the :guilabel:`Upload` button and :guilabel:`Share`.
```
IMO "select" is a bit too far from the tags part, making the sentence unclear. To avoid having two "and"s close together (which would be the result of reverting to "Go to the left column and select a workspace and possibly one or several tags"), I think breaking it into two separate sentences like I did in my suggestion works better.

Reply:
@Donapi I noticed that this paragraph seems to be missing after the last force-push (I think it should be after line 235?). Could it have been accidentally omitted during the rebase or while resolving conflicts?